src/AutorestSwift/View Models/ResponseViewModel.swift

```diff
@@ -73,13 +78,12 @@ struct ResponseViewModel {
             self.pagingNames = nil
             self.pagedElementClassName = nil
             if let objectType = schemaResponse?.schema.swiftType(optional: false) {
-                strategy = .body
+                self.strategy = getStrategy(objectType)
```

Comment:
I'm a little curious why we are alternating between `strategy` and `self.strategy`. Aren't we passing the code through `swiftlint autocorrect` and `swiftformat`?

Reply:
i think it is possible that i build using 'make install' (instead of XCode) more now. And the 'swiftlint autocorrect and swiftformat' are setup in Xcode workspace.'

Reply:
We should definitely have `make install` run it through the same process so as not to introduce random back-and-forth changes that aren't really part of the PR.

Reply:
yeah. I will address this in a separate pr.